packages/web/src/components/background-animations/MusicConfetti.tsx

```diff
@@ -1,81 +1,34 @@
+  ),
+  // Note
+  new Path2D(
+    'M3,13.0400072 L3,3.61346039 C3,3.16198653 3.28424981,2.76289841 3.70172501,2.62823505 L12.701725,0.0477059646 C13.3456556,-0.160004241 14,0.3365598 14,1.0329313 L14,12.9033651 C14,12.9179063 13.9997087,12.9323773 13.9991318,12.9467722 C13.9997094,12.9644586 14,12.9822021 14,13 C14,14.1045695 12.8807119,15 11.5,15 C10.1192881,15 9,14.1045695 9,13 C9,11.8954305 10.1192881,11 11.5,11 C11.6712329,11 11.838445,11.0137721 12,11.0400072 L12,3.61346039 L5,5.5488572 L5,14.9677884 C5,14.9726204 4.99996783,14.9774447 4.99990371,14.9822611 C4.99996786,14.988168 5,14.994081 5,15 C5,16.1045695 3.88071187,17 2.5,17 C1.11928813,17 0,16.1045695 0,15 C0,13.8954305 1.11928813,13 2.5,13 C2.67123292,13 2.83844503,13.0137721 3,13.0400072 Z'
+  ),
+  // Playlists
+  new Path2D(
+    'M5.46563786,16.9245466 C4.92388449,17.5744882 4.02157241,18 3,18 C1.34314575,18 0,16.8807119 0,15.5 C0,14.1192881 1.34314575,13 3,13 C3.35063542,13 3.68722107,13.0501285 4,13.1422548 L4,3.02055066 C4,2.49224061 4.31978104,2.02523181 4.78944063,1.86765013 L10.5394406,0.0558250276 C11.2638626,-0.187235311 12,0.393838806 12,1.20872555 C12,2.01398116 12,2.61792286 12,3.02055066 C12,3.62449236 11.4511634,4.01020322 11,4.12121212 C10.3508668,4.28093157 8.68420009,4.62436591 6,5.15151515 L6,16.0224658 C6,16.5009995 5.80514083,16.7960063 5.46563786,16.9245466 Z M13,6 L17,6 C17.5522847,6 18,6.44771525 18,7 C18,7.55228475 17.5522847,8 17,8 L13,8 C12.4477153,8 12,7.55228475 12,7 C12,6.44771525 12.4477153,6 13,6 Z M11,10 L17,10 C17.5522847,10 18,10.4477153 18,11 C18,11.5522847 17.5522847,12 17,12 L11,12 C10.4477153,12 10,11.5522847 10,11 C10,10.4477153 10.4477153,10 11,10 Z M11,14 L17,14 C17.5522847,14 18,14.4477153 18,15 C18,15.5522847 17.5522847,16 17,16 L11,16 C10.4477153,16 10,15.5522847 10,15 C10,14.4477153 10.4477153,14 11,14 Z'
+  )
 ]
 
-async function startConfettiAnimation(
-  canvasRef: HTMLCanvasElement,
-  recycle: boolean,
-  limit: number,
-  friction: number,
-  gravity: number,
-  rotate: number,
-  swing: number,
-  particleRate: number,
-  onCompletion: () => void,
-  isMatrix: boolean
-) {
-  if (!canvasRef) return
-  let images = null
-  if (!isMatrix) {
-    images = DEFAULT_IMAGES.map((icon) => {
-      const img = new Image()
-      img.src = icon
-      return img
-    })
-    await Promise.all(
-      images.map(
-        (img) =>
-          new Promise((resolve) => {
-            img.onload = () => {
-              resolve(true)
-            }
-          })
-      )
-    )
-  }
-
-  const confetti = new Confetti(
-    canvasRef,
-    images,
-    recycle,
-    limit,
-    friction,
-    gravity,
-    rotate,
-    swing,
-    particleRate,
-    onCompletion
-  )
-  confetti.run()
-  return confetti
-}
+const COLORS = ['#cc0fe0', '#7e1bcc']
```

Comment:
Just wondering if we should define these as constants somewhere in themes? Are they from the color palette?

Reply:
nice -- was able to pull them from the theme.